chore(browser agent): Fixed the numbering format and added a callout (#18772)
newrelic/docs-website: src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx

    // fetch 
    ƒ fetch() { [native code] }
    ```
    <Callout variant="tip">
    If you see this type of failure or you see a different response, AJAX instrumentation may be disabled or you may be using another script or library that is conflicting with New Relic instrumentation.
    </Callout>

  </Step>

  <Step>

>     ## Verify AJAX instrumentation is enabled

Reviewer: ```suggestion
    ## Verify AJAX instrumentation is enabled [#verify-instrum]
```

Author: Updated

> 

Reviewer: @adutta-newrelic I like how you broke up those long paragraphs! Still, I think you could make this a bit easier to follow by converting these into steps. What do you think about an approach like this?

 ## Verify AJAX instrumentation is enabled

 AJAX instrumentation is not included in the lite agent. 
 
 To verify Ajax instrumentation:
    
   1. Open your browser console dev tool and check the `newrelic.initializedAgents` object. This object contains an entry for every agent running on the page. There is typically only one agent on a page. 
   
   2. Inspect the initialized agent object for a `runtime` object. This object contains information about the running agent like `loaderType`. If `loaderType` is `lite`, AJAX instrumentation is not included in the agent being used.

   3. If the `loaderType` is not `lite`, you can also check the initialized agent for `config` and `features` objects. 
    
       * The `config` object will contain runtime configuration of the agent, including an `ajax` object with an `enabled` boolean.     
       * The `features` object should contain an entry for each feature initialized by the agent, including an `ajax` entry. 
    
   4. Verify this AJAX object contains a boolean `enabled` of `true` and `featAggregate` entry. 
    
    If any of these items are not true, the AJAX instrumentation may not have initialized due to a configuration change in New Relic on the browser entity or possibly a manual change to the `NREUM.init` object that was copy/pasted into the HTML.

    AJAX instrumentation is not included in the lite agent.

    To verify AJAX instrumentation:

    1. Open your browser console dev tool and check the `newrelic.initializedAgents` object. This object contains an entry for every agent running on the page. There is typically only one agent on a page.

    2. Inspect the initialized agent object for a `runtime` object. This object contains information about the running agent like `loaderType`. If `loaderType` is `lite`, AJAX instrumentation is not included in the agent being used.

    3. If the `loaderType` is not `lite`, you can also check the initialized agent for `config` and `features`
     objects.

        * The `config` object will contain runtime configuration of the agent, including an `ajax` object with an `enabled` boolean.
        * The `features` object should contain an entry for each feature initialized by the agent, including an `ajax` entry.

    4. Verify this AJAX object contains a boolean `enabled` of `true` and `featAggregate` entry.

> If any of these items are not true, the AJAX instrumentation may not have initialized due to a configuration change in New Relic on the browser entity or possibly a manual change to the NREUM.init object that was copy/pasted into the HTML.

Reviewer: ```suggestion
If any of these items are not true, the AJAX instrumentation may not have initialized due to a configuration change in New Relic on the browser entity or possibly a manual change to the `NREUM.init` object that was copied and pasted into the HTML.
```

    `newrelic.initializedAgents` is the accepted property to check for agent existence on a page, unless you are using an old or an unsupported agent version.
  </Callout>
  </Step>
  
  <Step>
  

>     ## Verify the deny list configuration

Reviewer: ```suggestion
    ## Verify the deny list configuration [#verify-deny-list]
```

Author: Updated

    The agent can be configured to ignore certain domains and paths when creating [AjaxRequest Events](/attribute-dictionary/?event=AjaxRequest).

    Use your browser's dev console to check the current AJAX deny list rules by running `newrelic.init`. Look for a property `ajax` containing an array called `deny_list`.

    If you see entries in the deny list that are unfamiliar or causing AJAX calls to be filtered, review the [Filter AjaxRequest Events](/docs/browser/new-relic-browser/configuration/filter-ajax-request-events/) documentation for further guidance.
  </Step>

  <Step>

>     ## Verify network access

Reviewer: ```suggestion
    ## Verify network access [#verify-network-access]
```

Author: Updated

    If the object is properly instrumented, try triggering an AJAX call in your application while monitoring network traffic in the browser's developer tools. Wait up to one minute, and look for a call to `bam.nr-data.net/jserrors` with an `xhr` parameter. If the call fails, check for network issues.

    If you don't see this call, if it fails with an error not related to network access, or if it succeeds but you still aren't seeing data, continue through the troubleshooting documentation.

  </Step>

  <Step>

>   If your requests use JSONP, see requirements and notes on functionality below:

Reviewer: @adutta-newrelic I think you could drop this sentence, and it wouldn't hurt the meaning of the section.

>   ## JSONP requirements

Reviewer: ```suggestion
  ## Review how JSONP affects requests [#jsonp]
```

Author: Updated

  
    If your requests use [JSONP](https://en.wikipedia.org/wiki/JSONP), these requests will not appear on the [AJAX UI page](/docs/browser/new-relic-browser/browser-pro-features/ajax-page-identifying-time-consuming-calls). 
    
    However, you can view them as assets within [session traces](/docs/browser/new-relic-browser/browser-pro-features/session-traces-exploring-webpages-life-cycle). 
    

>     If using SPA monitoring, you can view them on the <DNT>**Breakdown**</DNT> tab of the <DNT>**Page views**</DNT> page.

Reviewer: ```suggestion
    If you're using SPA monitoring, you can view them on the <DNT>**Breakdown**</DNT> tab of the <DNT>**Page views**</DNT> page.
```

    Requirements for JSONP to be recognized:

    * Each JSONP request must use a unique callback function. Most popular libraries (like jQuery) generate a unique callback function dynamically for each request.
    * The query string containing the callback function name must be named `"callback"` or `"cb"` in order to be recognized by New Relic. This is the default behavior in most popular libraries.

>   <Callout variant="tip">

Reviewer: @adutta-newrelic since this callout seems to apply to all the options in this troubleshooting guide, I'd move this to the left margin (not have it indented under the bulleted list items.

Author: Updated
